python/google/appengine/dist27/socket.py

```diff
@@ -317,8 +323,8 @@ def write(self, data):
         self._wbuf.append(data)
         self._wbuf_len += len(data)
         if (self._wbufsize == 0 or
-            self._wbufsize == 1 and '\n' in data or
-            self._wbuf_len >= self._wbufsize):
+            (self._wbufsize == 1 and '\n' in data) or
```

Comment:
Yeah, [this](http://bugs.python.org/issue879399) is when the parens were introduced so it looks like it was just a stylistic change.